libs/agno/agno/os/router.py

```diff
@@ -51,6 +62,71 @@
     from agno.os.app import AgentOS
 
 
+class WorkflowWebSocketManager:
+    """Manages WebSocket connections for workflow runs"""
+
+    active_connections: Dict[str, WebSocket]
+    workflow_connections: Dict[str, str]
+
+    def __init__(
+        self,
+        active_connections: Optional[Dict[str, WebSocket]] = None,
+        workflow_connections: Optional[Dict[str, str]] = None,
+    ):
+        # Store active connections: {connection_id: websocket}
+        self.active_connections: Dict[str, WebSocket] = active_connections or {}
+        # Store workflow-to-connection mapping: {run_id: connection_id}
+        self.workflow_connections: Dict[str, str] = workflow_connections or {}
```

Comment:
Why not have one dict, and the run_id **is** the connection ID? I.e. just map run_id to websocket.